admin/app/controllers/authenticated/certifications/configuration.js

```diff
@@ -0,0 +1,46 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+
+export default class ConfigurationController extends Controller {
+  @service store;
+  @service notifications;
+  competenceScoringConfiguration = '';
+  certificationScoringConfiguration = '';
+
+  @action
+  onCompetenceScoringConfigurationChange(event) {
+    this.competenceScoringConfiguration = event.target.value;
+  }
+
+  @action
+  async onCompetenceScoringConfigurationSubmit(event) {
+    event.preventDefault();
+    const adapter = this.store.adapterFor('scoring-configuration');
+    try {
+      await adapter.updateCompetenceScoringConfiguration(this.competenceScoringConfiguration);
+      this.competenceScoringConfiguration = '';
+      this.notifications.success('Configuration mise à jour');
+    } catch (e) {
+      this.notifications.error(`La config n'a pu être ajoutée: ${e}`);
+    }
+  }
+
+  @action
+  onCertificationScoringConfigurationChange(event) {
+    this.certificationScoringConfiguration = event.target.value;
+  }
+
+  @action
+  async onCertificationScoringConfigurationSubmit(event) {
+    event.preventDefault();
+    const adapter = this.store.adapterFor('scoring-configuration');
+    try {
+      await adapter.updateCertificationScoringConfiguration(this.certificationScoringConfiguration);
+      this.certificationScoringConfiguration = '';
+      this.notifications.success('Configuration mise à jour');
+    } catch (e) {
+      this.notifications.error(`La config n'a pu être ajoutée: ${e}`);
```

Comment:
```suggestion
    } catch (error) {
      this.notifications.error(`La configuration n'a pas pu être ajoutée: ${{error.errors[0].detail}}`);
```

<img width="415" alt="Capture d’écran 2024-03-20 à 10 30 24" src="https://github.com/1024pix/pix/assets/58915422/bdf2bf5e-dfe9-4e45-b277-c1ad05727b91">